Library solution

mortvicious opened the pull request:
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

ZachBaird commented:
Hello @mortvicious ! I replaced the spacing your pull request was deleting - the space is necessary for the list to render properly in markdown. There was a type as well with "Solution" which I fixed.

Is the repo for your library project [here](https://github.com/mortvicious/library)? Both your links point to your live project, and the first one usually should point to the repo itself (so students can see your code).

mortvicious commented:
> Hello @mortvicious ! I replaced the spacing your pull request was deleting - the space is necessary for the list to render properly in markdown. There was a type as well with "Solution" which I fixed.
> 
> Is the repo for your library project [here](https://github.com/mortvicious/library)? Both your links point to your live project, and the first one usually should point to the repo itself (so students can see your code).

Okay, got it! 

Yes, it is there. Should I fix it by myself? I saw a few people with this way of links order (and thought it's a bit strange to put two same links with different labels, but I'm not the person who ruin rules) and since that moment started to always order in this way

ZachBaird commented:
@mortvicious cool, no worries! 

I can just change it here. If you see other students doing that, feel free to let us know :) It's more helpful for folks to see both live product and the code that built it.
